docs/4-language-usage/4-control-structures/2-case-if-decode-nvl-nvl2-coalesce/g-4260.md

```diff
@@ -0,0 +1,34 @@
+# G-4260: Avoid inverting boolean conditions.
```

Comment:
I was unsure what the title meant with "inverting". The NOT? or the "not equal"? - After reading the reason and the examples it was clear. Why not change the title to "Avoid inverting boolean conditions with NOT"? In the end we want to avoid the NOT, right?